IMB-85: Add ban-only / DtR branch

mislam987 opened the pull request:
## What
- Add ban-only / Duty to Remove branch to the IMA form flow [IMB-85](https://collaboration.homeoffice.gov.uk/jira/browse/IMB-85)

## Why
- To allow the ban-only/DTR forking in the form
- To match the flow designed on Figma

## How
- Get the 'duty to remove alert' value from the session in /db/get-token.js
- Add conditional logic to index.js, using the session's 'duty to remove alert' value, to determine the flow
- Update /kube/hof-rds-api/deployment.yml with the latest Quay image ID that contains the new database column change

## Testing
- Tests passing locally

jamiecarterHO reviewed (approved):
Looks good but you will have to replace the hof-rds-api image with the latest if you want to deploy that change

mislam987 commented:
> Looks good but you will have to replace the hof-rds-api image with the latest if you want to deploy that change

Replaced the image ID in the deployment file, which also fixed the issue with the column name in the branch.

jamiecarterHO reviewed (approved):
LGTM, lets add a ticket to remove psql once database testing is complete.
